packages/flutter/lib/src/rendering/mouse_tracker.dart

```diff
@@ -238,8 +238,9 @@ class MouseTracker extends ChangeNotifier {
     assert(result != null);
     final LinkedHashMap<MouseTrackerAnnotation, Matrix4> annotations = LinkedHashMap<MouseTrackerAnnotation, Matrix4>();
     for (final HitTestEntry entry in result.path) {
-      if (entry.target is MouseTrackerAnnotation) {
-        annotations[entry.target as MouseTrackerAnnotation] = entry.transform!;
+      final Object target = entry.target;
```

Comment:
This was casting twice. For some reason if I type the LHS as HitTestTarget the cast doesn't take - perhaps because there is no common supertype (besides object) between HitTestEntry and MouseTrackerAnnotation